docs/machine_vars/index.md

```diff
@@ -37,7 +37,8 @@ display.
 If you want to use a machine variable in a slide player you can access
 it similarly to normal variables, you need to use the syntax
 `(machine|my_var_name)` where `my_var_name` obviously has to be replaced
-with your variable name.
+with your variable name. If you want to access the machine variable in a player, e.g. segment player you need to use this syntax
```

Comment:
This is a good callout, as folks often get confused. I think the explanation could be more explicit to clarify that the parenthesis-pipe syntax is when the variable _text_ is being used in a text widget or to create a text string, while the brackets-dot syntax is when the variable _value_ is being referenced or passed through.

Reply:
Valid point as well, have amended the text. Believe should be good now.